app.rb

```diff
@@ -0,0 +1,68 @@
+require 'sinatra/base'
+require './data_mapper_setup'
+require 'bcrypt'
+require 'sinatra/flash'
+
+ENV['RACK_ENV'] ||= 'development'
+
+class Chitter < Sinatra::Base
+  use Rack::MethodOverride
+  register Sinatra::Flash
+  enable :sessions
+  set :session_secret, 'super secret'
+
+  # When our Maker gets created at sign-up, their id is stored in a session. We then look for
+  # the instance of our Maker class that has an id equal to the session id, i.e. we retrieve
```

Comment:
Metrics/LineLength: Line is too long. [92/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)